src/sonic-yang-models/tests/yang_model_tests/tests/syslog.json

```diff
@@ -30,8 +30,7 @@
         "eStrKey": "InvalidValue"
     },
     "SYSLOG_SERVER_INVALID_IPADDR_TEST": {
-        "desc": "Load syslog server table with invalid ipv4 address as syslog server.",
-        "eStrKey": "InvalidValue"
```

Comment:
Why is the error removed? In case if this test is invalid, please remove the entire test

Reply:
fixed